src/calendarevent.cpp

```diff
@@ -130,7 +130,7 @@ CalendarEvent::Recur CalendarEvent::recur() const
 
 QDateTime CalendarEvent::recurEndDate() const
 {
-    return QDateTime(mData->recurEndDate);
+    return QDateTime(mData->recurEndDate.endOfDay());
```

Comment:
This and `.startOfDay()` were introduced later than Qt5.6. SailfishOS is still locked on 5.6 and thus all these changes must be inside `#if` macros on Qt version : /

Reply:
Yea, this. Reverting the QDateTime commit this built fine, though, so others are at least fine on the API availability sense.

Reply:
Good point!  I've wrapped each instance of this kind of change within a Qt version check like this:
```
#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
   // new version
#else
   // original version
#endif
```